addons/mail/static/src/discuss/core/common/discuss_core_common_service.js

```diff
@@ -18,6 +18,15 @@ export class DiscussCoreCommon {
     }
 
     setup() {
+        this.busService.addEventListener(
+            "connect",
+            () =>
+                this.store.imStatusTrackedPersonas.forEach((p) => {
+                    const model = p.type === "partner" ? "res.partner" : "mail.guest";
+                    this.busService.addChannel(`odoo-presence-${model}_${p.id}`);
```

Comment:
What's the point of the odoo prefix actually?
Just to avoid potential conflicts with custo in stable?

Reply:
Yes, to avoid conflicts. At least with the odoo prefix conflicts are limited to the odoo codebase.